### PR TITLE
Added Amex card icon src, updated payment icons configuration for AU & NZ

### PR DESCRIPTION
**Pull request by annieguo**
https://jira.just-eat.net/browse/MCC-752

NON - CONSUMER WEB | Only show the payment methods offered by Menulog (platform/tenant) in the website footer

##### How it was
screen shot 2018-12-04 at 11 31 35 am

##### How it will look like
screen shot 2018-12-04 at 11 35 56 am

##### Added
- Amex icon (not SafeKey) link in resource json file

##### Changed
- Amex icon for AU & NZ footer template json

##### Removed
- Paypal icon for NZ footer template json

### Browsers Tested

- [x] Chrome

### CODE REVIEW

**Comment by ashleynolan** [comment]
@annieguo Could you update the PR template with what has been tested & checked for this change?

**Comment by annieguo** [comment]
Hi @DamianMullins ,
package.json and change log are updated.
Cheers.

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-footer",
   "description": "Fozzie footer – Footer Component for Just Eat projects",
-  "version": "1.8.2",
+  "version": "1.8.3",
```
Comment (DamianMullins) [inline package.json]: Should bump the minor version here, rather than the patch, to become v1.9.0.

Comment (annieguo) [reply package.json]: @DamianMullins updated

Comment (DamianMullins) [reply package.json]: Hi @annieguo the version needs to be updated in both places, sorry!

Comment (annieguo) [reply package.json]: @DamianMullins 
Sorry, my bad, I meant to change both files. 
Now both updated.
Cheers.

**Comment by annieguo** [comment]
Hi @DamianMullins ,

How do we usually visually test a change like this? Or a change like this cannot be visually tested? As mentioned in the email and ui-fozzie slack channel, I haven't been able to do isolation test for AU & NZ tenants on this. Could you advise?

Thanks.

**Comment by annieguo** [comment]
@DamianMullins

Hi Damian,

I don't have permission to add reviewers, could you add another reviewer to this ticket in order to get it merged? Or can you merge it if it's ok?

**Comment by annieguo** [comment]
Hi @DamianMullins,

I've made some more changes on Visa & MasterCard icon for AU & NZ as well, I found the icons are not aligned with the rest of the pages, so I updated that to keep the consistency. 

Please review and see if it can be  merged.

Thank you.

Annie

**Comment by annieguo** [comment]
Hi @ashleynolan @DamianMullins,

I resolved the conflicts with master, please review again. 
Thank you.

Annie

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,19 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v2.0.0
```
Comment (ashleynolan) [inline CHANGELOG.md]: New version should be v1.10.0, as this is only a minor version change.

Also needs this to be updated in the `package.json`

**Comment by annieguo** [comment]
@ashleynolan 
Hi Ash,
The version number has been updated in CHANGELOG and package.json

Cheers.

Annie

**Comment by DamianMullins** [comment]
@annieguo This PR will need rebasing after a recent change, this means that you'll have to update the version to be v1.11.0.

**Comment by annieguo** [comment]
Hi @DamianMullins @ashleynolan @dandel10n,
I've just updated the version number, could you please review and see if it's ok to be merged.
Thank you.
Annie

**Pull request merged**